internal/admin/dashboard/templates/index.html

```
@@ -51,6 +51,63 @@ <h3 x-text="chartTitle()">Daily Token Usage</h3>
         </div>
         <p class="empty-state" x-show="daily.length === 0 && !loading && !authError">No usage data yet.</p>
     </div>
+
+    <!-- Contribution Calendar -->
+    <div class="contribution-calendar-section">
+        <div class="contribution-calendar-header">
+            <h3>Activity</h3>
+            <div class="usage-mode-toggle">
+                <button class="usage-mode-btn" :class="{active: calendarMode==='tokens'}" @click="toggleCalendarMode('tokens')">Tokens</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)